OpenRA.Mods.Common/Activities/Enter.cs

```diff
@@ -145,7 +146,10 @@ ReserveStatus TryReserveElseTryAlternateReserve(Actor self)
 						Target t = target;
 						if (!TryGetAlternateTarget(self, tries, ref t))
 							return ReserveStatus.TooFar;
-						if ((target.CenterPosition - self.CenterPosition).HorizontalLengthSquared <= (t.CenterPosition - self.CenterPosition).HorizontalLengthSquared)
+
+						var targetPosition = target.Positions.PositionClosestTo(self.CenterPosition);
+						var alternatePosition = t.Positions.PositionClosestTo(self.CenterPosition);
```

Comment:
Actually, it turns out that `targetCenter` has nothing to do with which position to target.  It is used to suppress repathing during group-transport orders.

I added a commit to replace that variable to make this clearer.